### PR TITLE
Election templates

### PR DESCRIPTION
**Pull request by JustFinchy**
Created template folder to allow future elections to be able to access past information. This will help allow expedite any elections that are held.

### CODE REVIEW

File: O3DE Elections/README.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # O3DE Election Guides and Resources
+
+|                           **Table of Contents**                            |
+|:--------------------------------------------------------------------------:|
+|      [Election Guide](./O3DE%20Elections%20Guide.md)      |
+|     [SIG-Release Template](./sig-templates/sig-release.md)     |
```
Comment (lmbr-pip) [inline O3DE Elections/README.md:6]: Considering adding some description as to what these are

File: O3DE Elections/sig-templates/sig-docs-community.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Hello all sig-docs-community:
```
Comment (lmbr-pip) [inline O3DE Elections/sig-templates/sig-docs-community.md:1]: nit: add link to sig-docs-community README

File: O3DE Elections/sig-templates/sig-docs-community.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+For the first year of O3DE, @sptramer and @FiniteStateGit have served as the chairpersons of the Documentation & Community Special Interest Group. Now it's time for our second election season, leading into a 6-month term, which is shorter than the original sig-docs-community term length. We think that electing new representatives at this time in the leadup to the 22.10 release and closing of the year of 2022 will help us grow the SIG and better understand our users and community. Due to the expertise of the chairs and SIG members during the first year of O3DE, we have focused very heavily on documentation, and encourage the nomination of persons who are more focused on building our community.
```
Comment (lmbr-pip) [inline O3DE Elections/sig-templates/sig-docs-community.md:3]: This doesn't read like a template (ie not ready to be reused)

Consider adding this as example content

File: O3DE Elections/sig-templates/sig-docs-community.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+The following things have changed since our last election:
+
+* Terms are now officially 6 months, meaning our next election will be held in mid-March of 2023. Chairpersons may relinquish their chairship at any point and for any reason, but nominees should try and commit to their full term.
+* Nominees are no longer required to be reviewers or maintainers, but reviewer/maintainer status is something nominees should highlight.
+* Nominees should have at least some familiarity with open source, and one of either technical writing or community management/building.
+* Nominees should provide a statement regarding their candidacy and what they see as important for the SIG to handle over the next 6-12 months. It's OK for a chair to set direction to continue after their term!
+* Chairs are not required to provide on-call coverage of any sort.
```
Comment (lmbr-pip) [inline O3DE Elections/sig-templates/sig-docs-community.md:11]: This should move into the expectations of the chair/co-chair roles section IMHO. Actually its repeated below, so consider removing

File: O3DE Elections/sig-templates/sig-docs-community.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## The chair / co-chair roles
```
Comment (lmbr-pip) [inline O3DE Elections/sig-templates/sig-docs-community.md:13]: To me this should be linked from the template as the some for of roles/responsibility doc is something the SIG should own separately from this template.

File: O3DE Elections/sig-templates/sig-docs-community.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+The chair and co-chair serve equivalent roles in the governance of the SIG and are only differentiated by title in that the highest vote-getter is the chair and the second-highest is the co-chair. The chair and co-chair are expected to govern together in an effective way and split their responsibilities to make sure that the SIG operates smoothly and has the availability of a chairperson at any time.
+
+Unless **distinctly required**, the term "chairperson" refers to either/both of the chair and co-chair. If a chair or co-chair is required to perform a specific responsibility for the SIG they will always be addressed by their official role title.
+
+Chairs are not required to provide "on-call" support of any kind as part of their chairship. The O3D Foundation is responsible for addressing any high-severity issues related to documentation or community which would impact the immediate operation of the O3DE project or may result in legal liability of the O3D Foundation.
+
+
+### Responsibilities
+
+* Schedule and proctor regular SIG meetings on a cadence to be determined by the SIG.
```
Comment (lmbr-pip) [inline O3DE Elections/sig-templates/sig-docs-community.md:24]: This should include:
* expectation for one of the chair(s) to be at the "SIG of SIG" TSC meeting (last meeting of the month)
* Maintenance of public roadmap items owned by SIG

File: O3DE Elections/sig-templates/sig-release.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+## The chair / co-chair roles
```
Comment (lmbr-pip) [inline O3DE Elections/sig-templates/sig-release.md:1]: IMHO, again, the SIG should own the roles responsibility part and the template links, or asks to copy the content

**Review by lmbr-pip — COMMENTED** [review]
Its unclear if these are "templates" as at least the docs one doesn't seem reusable.

But they are great examples for other SIGs to use.

IMHO what we should do is around roles and responsibilities of Chair(s)

a) Draft common Chair(s) responsibilities and add to https://github.com/o3de/community/blob/main/community-membership.md
    * Add a how do I become a Chair and link to this election guide

b) SIGs should have a list of their additions/deviations in their charter / separate chair_responsibility doc under their governance folder ie https://github.com/o3de/sig-simulation/tree/main/governance
     * All SIGS were given charters with a **Additional responsibilities of Chairs** section
    
We could work with docs or release to provide an example for other SIGs. Would be happy to help.

**Comment by lmbr-pip** [comment]
We could have a general SIG convernence doc under https://github.com/o3de/community/tree/main/sigs/docs and link to that

**Review by phuchau1989 — APPROVED** [review]

**Review by phuchau1989 — APPROVED** [review]

**Pull request closed without merging**